docs/DevGuide/CompilerLinkerErrors.md

```diff
@@ -59,10 +71,13 @@ brigand::list<brigand::integral_constant<int, 1>,
 brigand::integral_constant<int, 1> >,
 brigand::list<Tensor_detail::TensorIndexType<3ul, (UpLo)1, Frame::Inertial,
 (IndexType)0>, Tensor_detail::TensorIndexType<3ul, (UpLo)1, Frame::Inertial,
-(IndexType)0> > > const&)'
+(IndexType)0> > > const&)
 ```
 In order to make this easier to read, it is recommended to run the code through
-ClangFormat, which gives
+ClangFormat. This can be done by copying the linker output into an empty `cpp`
+file and running `clang-format -i EMPTY_FILE_WITH_LINKER_OUTPUT`. Keep in bind
```

Comment:
`Keep in bind` -> `Keep in mind`

Reply:
I swear, on Dvorak those keys are right next to each other ;)